meson.build

```diff
@@ -0,0 +1,42 @@
+project('scs', 'c')
+
+py = import('python').find_installation(pure: false)
+
+# This uses the path as is, and avoids running the interpreter.
+incdir_numpy = run_command(py,
+    [
+      '-c',
+      '''import os
+import numpy as np
+incdir = np.get_include()
+print(incdir)
+'''
+    ],
+    check: true
+  ).stdout().strip()
+
+incdir = include_directories('scs/include')
+pyincdir = include_directories('src')
+linsysinc = include_directories('scs/linsys')
+indirectinc = include_directories('scs/linsys/cpu/indirect')
```

Comment:
You don't have to do it like this - you're only using these in one place, so you can do this directly inside `py.extension_module`:
```meson
    include_directories : [
        'scs',
        'scs/include',
        'scs/linsys',
        'scs/linsys/cpu/indirect',
    ]